lib/trento/application/integration/checks/adapter/runner.ex

```diff
@@ -0,0 +1,52 @@
+defmodule Trento.Integration.Checks.Runner do
+  @moduledoc """
+  Trento runner integration adapter
+  """
+
+  @behaviour Trento.Integration.Checks.Gen
+
+  require Logger
+
```

Comment:
`@runner_url Application Application.compile_env!(:trento, __MODULE__)[:runner_url]`

Reply:
Is it not better to have:
```
defp runner_url,
    do: Application.fetch_env!(:trento, __MODULE__)[:runner_url]
```

As this value must be obtained on `runtime` in production from a ENV variable

Reply:
yes my bad, this is the way!